groups/createElements.js

```diff
@@ -161,13 +173,17 @@ const createGroupCreationModal = (onClose = () => {}, onSubmit = () => {}) => {
 
   form.onsubmit = (e) => {
     e.preventDefault();
-    const inputValue = getInput();
-    if (inputValue.length === 0 || inputValue.includes('group')) {
+    const title = getTitle();
+    const description = descriptionField.value.trim();
+    if (title.length === 0 || title.includes('group')) {
       throw new Error('Invalid group name');
     }
 
```

Comment:
Improve error messages for better clarity. Instead of generic messages like `Invalid group name,` provide specific reasons.